sklearn/decomposition/kernel_pca.py

```diff
@@ -81,6 +81,11 @@ class KernelPCA(BaseEstimator, TransformerMixin):
         A pseudo random number generator used for the initialization of the
         residuals when eigen_solver == 'arpack'.
 
+    copy : boolean, default=True
```

Comment:
Some of the `sklearn.decomposition` (e.g. PCA) already use `copy=True`. I don't think it's a good idea to fragment the package. Or should we refactor all of those to `copy_X`?